react.js

```diff
@@ -18,6 +18,12 @@ module.exports = {
     ]
   },
   'settings': {
+    'react': {
+      'version': 'detect',
+    },
+    'linkComponents': [
```

Comment:
Can you explain this?

Reply:
Ah, I see the docs you linked to. Do we not need `"Hyperlink"?

Reply:
Yeah, I think that's just another possible link component name. Ala https://www.npmjs.com/package/react-native-hyperlink?

Reply:
ohhh, nice catch! Cool